### PR TITLE
Docs: updating "Modifying the schema" for `aiida-core==1.0.0`

### LINKED ISSUE #3176: Check documentation for "for AiiDA developers/Modifying the schema"
Check if the section is updated to AiiDA 1.0.

https://aiida.readthedocs.io/projects/aiida-core/en/latest/developer_guide/core/modifying_the_schema.html

### PR DESCRIPTION
**Pull request by sphuber**
Fixes #3176

### CODE REVIEW

File: docs/source/developer_guide/core/modifying_the_schema.rst [hunk 1]
```diff
@@ -3,220 +3,192 @@ Modifying the schema
...
+            statement = text('\n'.join(forward_sql))
+            conn.execute(statement)
+        def downgrade():
+            conn = op.get_bind()
+            statement = text('\n'.join(reverse_sql))
+            conn.execute(statement)
+
+    If you want to learn more about the migration operations, you can have a look at the Alembic documentation.
+
+ 4. Your database will be automatically migrated to the latest revision as soon as you run your first verdi command.
```
Comment (giovannipizzi) [inline docs/source/developer_guide/core/modifying_the_schema.rst]: Is this still true?

Comment (sphuber) [reply docs/source/developer_guide/core/modifying_the_schema.rst]: Nope, you are right. It now follows the same paradigm as Django and simply prompts a command to be executed

Comment (sphuber) [reply docs/source/developer_guide/core/modifying_the_schema.rst]: I also realize now that the Django section still instructs to use the Django manage tool directly, i.e. `python aiida/backends/djsite/manage.py migrate`. I think people, even developers, should use `verdi database migrate` instead. Should I homogenize the two sections to reflect this?

Comment (giovannipizzi) [reply docs/source/developer_guide/core/modifying_the_schema.rst]: Sure, if you can it would be great!

**Pull request merged**